frontend/src/metabase/metadata/pages/DataModel/components/TablePicker/Results.module.css

```diff
@@ -16,11 +16,18 @@
     outline: 2px solid var(--mb-color-focus);
   }
 
-  &.active,
   &:hover {
     background-color: var(--mb-color-brand-lighter);
+  }
+
+  &.active {
     color: var(--mb-color-brand);
 
+    &,
+    &:hover {
```

Comment:
This selector does not make sense just from reading it. If `&` already has the `background-color` why do we need to apply it when `:hover`-ing too?

Reply:
ef8a1e2f9c7871d4fa8011a459805e20145c6f6d